testing/gimmeproj/main.go

```diff
@@ -33,10 +34,12 @@ import (
 var (
 	metaProject = flag.String("project", "", "Meta-project that manages the pool.")
 	format      = flag.String("output", "", "Output format for selected operations. Options include: list")
+	waitTime    = flag.Duration("timeout", 30*time.Minute, "maximum wait time for leasing a project")
```

Comment:
question: Where does 30 min come from? Should this be a smaller number? The larger the number the more resources we are locking down for resource polling.

Reply:
This was arbitrarily chosen, based on my personal experience that the shortest test runs seem to take about 15-20m. The longest ones go for 2h+, so that's basically the range for this option.

the polling operation is pretty quick (one datastore API call), so i stuck toward the low end. we can always specify a different time with this flag so 🤷🏼